tests/integration/test_fim/test_registry/test_checks/test_check_all.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+
+from wazuh_testing.fim import CHECK_GROUP, CHECK_MTIME, CHECK_OWNER, CHECK_PERM, \
+                              CHECK_SHA256SUM, CHECK_SIZE, CHECK_SUM, CHECK_ALL, \
+                              CHECK_TYPE, LOG_FILE_PATH, REQUIRED_REG_VALUE_ATTRIBUTES, KEY_WOW64_32KEY, \
+                              KEY_WOW64_64KEY, REQUIRED_REG_KEY_ATTRIBUTES, generate_params, registry_value_cud, \
+                              registry_key_cud
+
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing import global_parameters
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
+sub_key_1 = "SOFTWARE\\Classes\\testkey"
+sub_key_2 = "SOFTWARE\\testkey"
+sub_key_3 = "SOFTWARE\\testkey3"
+sub_key_4 = "SOFTWARE\\testkey4"
+sub_key_5 = "SOFTWARE\\testkey5"
+sub_key_6 = "SOFTWARE\\testkey6"
+
+
+test_regs = [os.path.join(key, sub_key_1),
+             os.path.join(key, sub_key_2),
+             os.path.join(key, sub_key_3),
+             os.path.join(key, sub_key_4),
+             os.path.join(key, sub_key_5),
+             os.path.join(key, sub_key_6)
+             ]
+
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+key_all_attrs = REQUIRED_REG_KEY_ATTRIBUTES[CHECK_ALL]
+value_all_attrs = REQUIRED_REG_VALUE_ATTRIBUTES[CHECK_ALL]
+
+params_list = [(key, sub_key_1, key_all_attrs, value_all_attrs - REQUIRED_REG_VALUE_ATTRIBUTES[CHECK_SUM]),
+               (key, sub_key_2, key_all_attrs, value_all_attrs - {CHECK_SHA256SUM}),
+               (key, sub_key_3, key_all_attrs, value_all_attrs - {CHECK_TYPE}),
+               (key, sub_key_4, key_all_attrs, value_all_attrs - {CHECK_SIZE}),
+               (key, sub_key_5, key_all_attrs - {CHECK_MTIME}, value_all_attrs),
+               (key, sub_key_6, key_all_attrs - {CHECK_OWNER} - {CHECK_GROUP} - {CHECK_PERM}, value_all_attrs)
+               ]
```

Comment:
This variable is used once as part of a parametrization of a test, could we remove the variable and place this options as part of the parametrization?